prov/util/src/util_attr.c

```diff
@@ -134,6 +134,15 @@ static int ofix_alter_layer_info(const struct fi_provider *prov,
 	if (!(*base_info = fi_allocinfo()))
 		return -FI_ENOMEM;
 
+	if (layer_info && layer_info->fabric_attr) {
+		if (layer_info->fabric_attr->name) {
+			(*base_info)->fabric_attr->name = strdup(layer_info->fabric_attr->name);
+			if (!(*base_info)->fabric_attr->name)
+				goto err;
+		}
+		(*base_info)->fabric_attr->api_version = layer_info->fabric_attr->api_version;
+	}
```

Comment:
PR #2808 has these values set by the framework. The app's api_version is a requirement on the util provider.  If the util provider has an api_version requirement on the core provider, it should specify that to the core separately.  It need not be the same as the api_version requested by the app.  I.e. the util provider could support any api_version, but require that the core support version >= 1.5.